include/matx/operators/matmul.h

```diff
@@ -113,7 +114,17 @@ namespace matx
 
         template <typename Out, typename Executor>
         void Exec(Out &&out, Executor &&ex) const {
-          if constexpr (!std::is_same_v<PermDims, no_permute_t>) {
+          // Perform SpMM or otherwise GEMM.
+          static_assert(!is_sparse_tensor_v<OpB>, "sparse rhs not implemented");
```

Comment:
what's the reason this can't be done?

Reply:
We have a direct SpMM, not MSpM in cuSPARSE